### ISSUE TITLE
I have tried this solution before, but support is very POOR and hard to understand

### ISSUE THREAD

**Issue by mach37** [optimajet/WorkflowEngine.NET#1]
I'm giving feedback, because I worked on this solution over a yr ago, when it used to be on Codeplex. Because my lead chose to adopt it in the hopes of saving time. We were referred to it by a 3rd party.

1) The UI is very buggy, hard to understand.
2) The commands and architecture is hard to follow
3) New products are introduced without the first version full maturing, so anyone who tries to use these components is stuck with semi-functional pieces.
4) This is not a purely open source model, when you start implementing you will realize lot of components are closed and need a purchase. You need to clearly disclose..
5) Only one developer behind the solution (and can be abrasive on responses at times)
6) Most important, you need to be service oriented and nicer to your customers
7) Nservice bus was an easier migration IMHO, I'm doing this in my new project and so much easier.
8) the Javascript is not shared, and obfuscated, so you can not debug any issues!

Just my two cents, hopefully you take it constructively and fix these issues, please be nice to your customers!

**Comment by dmelnikov** [comment]
Thanks for your feedback.

What version do you use? 
Could tell me what is your company?

**Comment by papyr** [comment]
Can you share any of your fixes. I did notice the JS has now been unobfuscated, I was trying to fix issues in the designer, but there were no comment.

Do you support auto - upgrades, what happens if we write custom code, and new version comes out?

**Comment by dmelnikov** [comment]
Dear Papyr.

Right now, we provide customization through js-class WorkflowConstaints.

If your company have (or had) the enterprise (or company) license of WFE then you may connect me via email or skype and we discuss about how to adapted the workflow designer for your requirements that your changes not lost and will be actual after next update of WFE.
